### DISCUSSION TITLE
No alignment without linear spectrograms

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/TTS#50] [Archived]
<USER>>>> geneing<br/> <DATE> [May 21, 2019, 4:49pm]<br/> <br/> I need help understanding something.<br/> <br/> I removed the linear spectrogram part of the loss function, along with<br/> postnet that generates it. I didn't need the linear spectrogram for the<br/> vocoder and removing the linear spectrogram part save a LOT of GPU<br/> memory during training. However, the reduced model doesn't produce<br/> reasonable attention even after 50K steps. For the full model, attention<br/> was reasonable after only a few thousand steps.<br/> <br/> Why isn't mel spectrogram part of the loss not enough to train the<br/> attention?<br/> <br/> [This is an archived TTS discussion thread from discourse.mozilla.org/t/no-alignment-without-linear-spectrograms]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> erogol<br/> <DATE> [May 23, 2019, 12:13am]<br/> <br/> could you check gradient norms between these two runs? Maybe , the scale<br/> of loss value has been changed and it requires a new learning rate. If<br/> you have tensorboard files, you can also share them.<br/> <br/> [Archived Post]<br/>
